src/Model/Term.php

```diff
@@ -194,15 +194,9 @@ protected function init() {
 					return $queue;
 				},
 				'uri'                      => function () {
```

Comment:
Replacing the callback with the following _should_ do the trick (I'm not 100% sure the slash situation with `WP_Site->$path`):

```php
'uri'  => function () {
	$link = $this->link;

	if ( empty( $link ) ) {
		return null;
	}

	$parsed = wp_parse_url( $link );

	if ( empty( $parsed ) ) {
		return null;
	}

	// @todo We should probably check the the URL is for the current site before making it relative.

	$path  = $parsed['path'] ?? '';
	$query = isset( $parsed['query'] ) ? ( '?' . $parsed['query'] ) : '';

	// Strip out multisite directory.
	if ( ! empty( $path ) && is_multisite() ) {
		$site = get_site();

		if ( $site instanceof \WP_Site ) {
			$path = str_ireplace( ltrim( untrailingslashit( $site->path ), '/' ), '', $path );
		}
	}

	return  trim( $path . $query );
},
```